rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -0,0 +1,382 @@
+            PRIMARY KEY(identifier, child_asset_id)
+        );
+        """)
+        cursor.execute("""
+        CREATE TABLE IF NOT EXISTS user_owned_assets (
+            asset_id VARCHAR[24] NOT NULL PRIMARY KEY,
+            FOREIGN KEY(asset_id) REFERENCES assets(identifier) ON UPDATE CASCADE
+        );
+        """)
+        cursor.execute("""
+        CREATE TABLE IF NOT EXISTS underlying_tokens_list (
+            identifier TEXT NOT NULL,
+            weight TEXT NOT NULL,
+            parent_token_entry TEXT NOT NULL,
+            FOREIGN KEY(parent_token_entry) REFERENCES evm_tokens(identifier)
+                ON DELETE CASCADE ON UPDATE CASCADE
+            FOREIGN KEY(identifier) REFERENCES evm_tokens(identifier) ON UPDATE CASCADE
+            PRIMARY KEY(identifier, parent_token_entry)
+        );
+        """)
+        cursor.execute("""
+        CREATE TABLE IF NOT EXISTS price_history (
+            from_asset TEXT NOT NULL COLLATE NOCASE,
+            to_asset TEXT NOT NULL COLLATE NOCASE,
+            source_type CHAR(1) NOT NULL DEFAULT('A') REFERENCES price_history_source_types(type),
+            timestamp INTEGER NOT NULL,
+            price TEXT NOT NULL,
+            FOREIGN KEY(from_asset) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
+            FOREIGN KEY(to_asset) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
+            PRIMARY KEY(from_asset, to_asset, source_type, timestamp)
+        );
+        """)  # noqa: E501
+        cursor.execute("""
+        CREATE TABLE IF NOT EXISTS asset_collection_properties(
+            identifier TEXT NOT NULL PRIMARY KEY,
+            name TEXT NOT NULL,
+            symbol TEXT NOT NULL,
+            FOREIGN KEY(identifier) REFERENCES multiasset_collector(identifier) ON UPDATE CASCADE ON DELETE CASCADE
+        );
+        """)  # noqa: E501
+        cursor.execute("""
+        CREATE TABLE IF NOT EXISTS custom_assets(
+            identifier INTEGER NOT NULL PRIMARY KEY,
+            name TEXT NOT NULL,
+            symbol TEXT,
+            notes TEXT,
+            type TEXT
+        );
+        """)
+
+        cursor.executescript('PRAGMA foreign_keys=off;')
+        cursor.executemany(COMMON_ASSETS_INSERT, common_asset_details)
+        cursor.executemany(COMMON_ASSETS_INSERT, common_asset_details_others)
+        cursor.executemany(ASSETS_INSERT, assets_tuple)
+        cursor.executemany(ASSETS_INSERT, assets_tuple_others)
+        cursor.executemany(OWNED_ASSETS_INSERT, owned_assets)
+        cursor.executemany(PRICES_INSERT, updated_prices)
+        cursor.executescript('PRAGMA foreign_keys=on;')
+        cursor.executemany(EVM_TOKEN_INSERT, evm_tuples)
+        cursor.executemany(UNDERLYING_TOKEN_INSERT, mappings)
+
+        dir_path = Path(__file__).resolve().parent.parent.parent
+        with open(dir_path / 'data' / 'globaldb_v2_v3_assets.sql', 'r') as f:
```

Comment:
Please add a docstring explaining what this file contains and what you are doing here. Also this file should be added in the setup.py and in the other file what's its name so that it's included with the binaries.

Reply:
Yes. Had in mind to add it and then forgot. This file contains the EVM version of the assets that are currently in the database and are not EVM (matic tokens, Otimism tokens, etc) + their variants in other chains. The info I got it using your script

Reply:
Good I guessed it was this but just wanted to confirm. Will add an explainer comment.